pkg/network/setup/netpod/netpod_test.go

```diff
@@ -1675,7 +1682,7 @@ func (c *tempCacheCreator) New(filePath string) *cache.Cache {
 	return cache.NewCustomCache(filePath, kfs.NewWithRootPath(c.tmpDir))
 }
 
-func expectedDHCPConfig(podIfaceCIDR, podIfaceMAC, defaultGW, staticRouteDst string) (*cache.DHCPConfig, error) {
+func expectedDHCPConfig(podIfaceCIDR, podIfaceMAC, defaultGW, staticRouteDst, staticRouteToWiderSubnet string) (*cache.DHCPConfig, error) {
```

Comment:
IMO `staticRouteDst` should have been made a slice.

Reply:
Since `staticRouteDst` and `staticRouteToWiderSubnet` are treated differently (one has a gateway and the other does not), we cannot use a slice here.

We can consider extracting the routes building to the caller in a follow up.